src/app/shared/helper-pipes/kpi-metrics-list.pipe.ts

```diff
@@ -9,7 +9,7 @@ import * as _ from 'lodash';
 export class KpiMetricsListPipe implements PipeTransform {
 
   transform(optionValue: KeyPerformanceIndicatorValue): Array<KeyPerformanceMetric> {
```

Comment:
it seems there is a mismatch with return type Array<KeyPerformanceMetric>, but keyPerformanceMetrics is Array<KeyPerformanceMetricOption>

Reply:
Fixed thanks